source/frontend/src/models/generate/GenerateOwner.ts

```diff
@@ -3,6 +3,7 @@ import { phoneFormatter } from '@/utils/formUtils';
 
 import { Api_GenerateAddress } from './GenerateAddress';
 export class Api_GenerateOwner {
+  owner_row: number;
```

Comment:
what this used for?

Reply:
Sadly, this is used to be able to conditionally render "TO:" John Doe on the first owner but use "AND:" on the rest. Here is the logic in the template. I guess I could rename to owner_index if it makes it more clear...

![image](https://github.com/bcgov/PSP/assets/24322224/89877757-2703-433f-8363-a02d4f76ea15)

This is done to comply with the following acceptance criteria on the template:
"// List all the owners with first owner name Prefixed with ‘TO:’ and subsequent with ‘AND:’, ...."

Reply:
I tried using Carbone logic for this but could not get it working - that is determining the current index of the array of owners being rendered by carbone

Reply:
on carbone you can refer to the index as well

Reply:
I tried using this `{d.file.owners[i]:count():ifEQ(1)}` but that didn't work. The count "prints" the row number but it doesn't pass it to the next formatter ifEQ()

Reply:
I'm modifying GenerateOwner to instead take an "owner_prefix" (empty by default) that way I can set the prefix just for expropriation forms. Don't like the index much

Reply:
Okay - found a way to do it in carbone. Will remove the owner_row